TEST_WORKFLOW.md

```diff
@@ -0,0 +1,136 @@
+X-Road Tests development
+# Workflow Policy
+
+v1.0
+
+## 1	General
+
+1.1	This document establishes workflow policy for joint development of X-Road software. Workflow Policy strives to serve as a handbook for all practical aspects of development.
+
+1.2	Workflow is understood as systematic arrangement of work, a complex of processes, practices, roles and responsibilities, communication patterns, and artefacts.
+
+1.3	Goals of the workflow policy are to
+- establish productive and secure collaborative open source working environment
+- assure production of high quality software
+- avoid duplication of effort, facilitate re-use of software
+- transparency and openness
+- broader community of developers
+- use of software development best practice
+- clear communication among Partners as well as other stakeholders in the X-Road development process
```

Comment:
...stakeholders in the X-Road _test_ development process

Reply:
done